Archives/VolFile.cpp

```diff
@@ -19,6 +19,8 @@ namespace Archives
 
 	VolFile::VolFile(const char *fileName) : ArchiveFile(fileName), archiveFileReader(fileName)
 	{
+		static_assert(sizeof(SectionHeader) == 8, "SectionHeader not of required size");
```

Comment:
This should be placed right after the `struct SectionHeader` definition. It doesn't need to be in a function, so it makes sense to put it right after the struct it applies to.